corona-warn-app/cwa-app-android
# PPA exposure risk metadata (EXPOSUREAPP-5863)

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5863

The old implementation would set the risk level to 0 and the date to -1 whenever the latest result was failed. 

According to the tech spec, it was expected to only have risk levels Increased and low otherwise, which corresponds to the last successful calculation. With this PR we provide data per risk calculation which corresponds to the ppa metadata data points. The combined result is not of interest.

https://github.com/corona-warn-app/cwa-app-tech-spec/blob/main/docs/spec/data-donation-client-ppa.md#exposure-risk-metadata

How to test:

Create a successful and a failed calculation and check that the ppa data contains the data from the successful calculation
